helm-system-packages-brew.el

```diff
@@ -0,0 +1,214 @@
+  ;; TODO: We should only create the buffer if it does not already exist.
+  ;; On the other hand, we need to be able to override the package list.
+  ;; (unless (helm-candidate-buffer) ...
+  (helm-init-candidates-in-buffer
+      'global
+    (if helm-system-packages-show-descriptions-p
+        helm-system-packages--descriptions
+      helm-system-packages--names)))
+
+(defun helm-system-packages-brew-refresh ()
+  "Refresh the package list."
+  (interactive)
+  (setq helm-system-packages--source-name "brew source")
+  (setq helm-system-packages-column-width
+        (or helm-system-packages-brew-column-width
+            helm-system-packages-column-width))
+    (let ((res (helm-system-packages-brew-cache)))
+      (setq helm-system-packages--names (car res)
+            helm-system-packages--descriptions (cdr res)))
+    (setq helm-system-packages--display-lists nil))
+
+(defun helm-system-packages-brew-info (_candidate)
+  "Print information about the selected packages.
+
+With prefix argument, insert the output at point.
+Otherwise display in `helm-system-packages-buffer'."
+  (let* ((descriptions (json-read-from-string (with-temp-buffer
+						(apply 'call-process "brew" nil t nil "info" "--json=v1" (helm-marked-candidates)) 
+						(buffer-string))))
+	 desc-list
+	 pkg-desc-alist
+	 str
+	 pkg
+	 (i 0))
+    (dolist (pkg (helm-marked-candidates) nil)
+      (setq pkg-desc-alist (aref descriptions i))
+      (setq str (concat "* Description: " (alist-get 'desc pkg-desc-alist) "\n"
+	      "* Version: "(alist-get 'stable (alist-get 'versions pkg-desc-alist)) "\n"
+	      "* URL: "(alist-get 'homepage pkg-desc-alist) "\n"  "\n"
+	      "* Dependencies:\n" "   " (mapconcat 'identity (alist-get 'dependencies pkg-desc-alist) "\n   ") "\n\n"
+	      "* Optional dependencies:\n" "   "(mapconcat 'identity (alist-get 'optional_dependencies pkg-desc-alist) "\n   ") "\n\n"
+	      "* Options:\n" (mapconcat (lambda (pkg-option)
+					(concat (alist-get 'option pkg-option)  "\n"
+						"    " (alist-get 'description pkg-option)  "\n"))
+				      (alist-get 'options pkg-desc-alist) "\n")
+	      "\n\n"
+	      "* Caveats: " (alist-get 'caveats pkg-desc-alist) "\n"))
+     (add-to-list 'desc-list `(uninstalled (,pkg . ,str)))
+     (setq i (+ i 1)))
+    (helm-system-packages-show-information desc-list)))
+
+(defun helm-system-package-brew-browse-url (_candidate)
+   (let* ((descriptions (json-read-from-string (with-temp-buffer
+						(apply 'call-process "brew" nil t nil "info" "--json=v1" (helm-marked-candidates)) 
+						(buffer-string)))))
+    (helm-system-packages-browse-url (mapcar (lambda(pkg)
+					       (alist-get 'homepage pkg))
+					     descriptions))))
+
+(defun helm-system-package-brew-link-app (_candidate)
+  (helm-system-packages-brew-run  "brew" "link"))
+
+(defun helm-system-package-brew-unlink-app (_candidate)
+  (helm-system-packages-brew-run  "brew" "unlink"))
+
+
+
+(defun helm-system-packages-brew-run (command &rest args)
```

Comment:
Just to be sure: This is the same as `helm-system-packages-call-as-root` except that it does not use `sudo`, right?
If so, mention it in the documentation.  Also `helm-system-packages-call-as-root` has undergone some changes, you might want to reflect them here.